discovery/README.md

```diff
@@ -12,6 +12,9 @@ Docker Swarm comes with multiple Discovery backends.
 
 ### Hosted Discovery with Docker Hub
 
+This service is hosted by Docker Hub and requires all nodes to have
```

Comment:
@vieux "This service" is ambiguous -- do you mean Docker Swarm?  We haven't really said it was a service anywhere else.  So, maybe  "The Docker Swarm image is hosted on Docker Hub. Using the Swarm image requires all the nodes to have internet access"